lib/cronitor.rb

```diff
@@ -34,7 +34,7 @@ def initialize(token: nil, opts: {}, code: nil)
   def create
     response = Unirest.post(
       "#{API_URL}/monitors",
-      headers: default_headers.merge 'Content-Type' => 'application/json',
+      headers: default_headers.merge('Content-Type': 'application/json'),
```

Comment:
This will make `'Content-Type'` into a symbol. I don't think you want that. E.g.:
```ruby
[4] pry(main)> default_headers = {}
=> {}
[5] pry(main)> default_headers.merge('Content-Type': 'application/json')
=> {:"Content-Type"=>"application/json"}
```
Also, this is just a stylistic change and probably shouldn't be included in this pull request.